src/compositor/lipstickcompositor.cpp

```diff
@@ -702,25 +733,140 @@ void LipstickCompositor::clipboardDataChanged()
         overrideSelection(const_cast<QMimeData *>(mimeData));
 }
 
-void LipstickCompositor::setUpdatesEnabled(bool enabled)
+bool LipstickCompositor::ambientSupported() const
+{
+    void* ambientMode = QGuiApplication::platformNativeInterface()->nativeResourceForIntegration("AmbientSupported");
```

Comment:
Just return QGuiApplication::platformNativeInterface()->nativeResourceForIntegration("AmbientSupported"); should work right?

Reply:
Tested your suggestion, that works fine. Ill create a new pr eventually for this :)